### ISSUE TITLE
Build with "-Werror" on all platforms

### ISSUE THREAD

**Issue by rgorosito** [d-ronin/dRonin#2244]
* Seen in version 20180729: 

The flag **-Werror** only was enabled on Linux, but it broken the build with modern versions of gcc. To solve this issue, is required to update de Qt version an fix all warnings

**Comment by tracernz** [comment]
Yes. I think the best course would be to move to Qt 5.12.2 (current LTS). There is a bit of background work that needs to happen for that to occur though. Maybe it's better to disable Werror for now?
